bom/pom.xml

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+       <groupId>org.apache.activemq</groupId>
+       <artifactId>activemq-parent</artifactId>
+       <version>6.1.1-SNAPSHOT</version>
```

Comment:
It does not make sense to add this in a point release. Adding a new module should be done for 6.2.0 at the earliest and not something like 6.1.1

Reply:
Not a problem, will rebase once JB releases 6.1.1 and we update main to 6.2.0-SNAPSHOT

Reply:
We should go ahead and create a 6.1.x branch and bump main to 6.2.0-SNAPSHOT which is what we typically do.